conf/machine/include/imx8x-mek.inc

```diff
@@ -54,6 +54,11 @@ UBOOT_CONFIG[fspi] = "${UBOOT_CONFIG_BASENAME}_fspi_defconfig"
 
 IMX_BOOT_SEEK = "32"
 
+# This machine is not supported by u-boot-fslc, so we force it to use
+# u-boot-imx here.
+IMX_DEFAULT_BOOTLOADER = "u-boot-imx"
```

Comment:
Why no overrides here like the other `imx8*.inc`?

Reply:
This is not supported by `u-boot-fslc` yet so we force it to use the same in any case.